nucypher/cli/commands/ursula.py

```diff
@@ -170,8 +169,8 @@ def generate_config(self, emitter, config_root, force, key_material):
         if self.dev:
             raise RuntimeError('Persistent configurations cannot be created in development mode.')
 
-        if (not self.operator_address) and not self.federated_only:
-            prompt = "Select operator account"
+        if not self.operator_address:
```

Comment:
Should `force` be considered here and other places with undefined information i.e. raise and don't prompt if force is true?

Reply:
Hmm, I suppose that's ambiguous usage.  If there's no value provided via `--operator-address` and `--force` is passed we'd need to crash to handle properly.  It looks like this is ore-existing behavior though so let's put it off into a follow-up PR?

Reply:
Sure 👍 , but let's get an issue filed before we merge - don't want to lose track of it.

Reply:
https://github.com/nucypher/nucypher/issues/3039